### PR TITLE
Fix organization errors 

### PR DESCRIPTION
**Pull request by fab-toc**

### CODE REVIEW

File: app/(modals)/organizations/[id].tsx [hunk 1]
```diff
@@ -22,19 +23,23 @@ export default function AssoIdPage() {
...
+        <ActivityIndicator />
+      ) : (
+        <Profile
+          avatar={data?.organization.logo_url}
+          handleRefresh={handleRefresh}
+          isRefreshing={isRefreshing}
+          posts={posts}
+          postsAreLoading={postsAreLoading}
+          setSize={setSize}
+          size={size}
```
Comment (aaalloc) [inline app/(modals)/organizations/[id].tsx:36]: size will always have a value ?

File: app/(modals)/organizations/_features/organization-profile.schema.ts [hunk 2]
```diff
@@ -10,14 +10,14 @@ const organizationSchema = z.object({
   instagram_link: z.string().url(),
   discord_link: z.string().url(),
   email: z.string().email(),
-  logo_url: z.string().url(),
+  logo_url: z.string().url().nullable(),
```
Comment (aaalloc) [inline app/(modals)/organizations/_features/organization-profile.schema.ts:13]: Maybe .optional is what we want https://zod.dev/?id=optional but I'm not to familar with zod

File: app/(modals)/organizations/[id].tsx (hunk 1 above)
Comment (fab-toc) [reply app/(modals)/organizations/[id].tsx:36]: yes, it's for pagination and infinite scrolling, same as in posts

File: app/(modals)/organizations/_features/organization-profile.schema.ts (hunk 2 above)
Comment (fab-toc) [reply app/(modals)/organizations/_features/organization-profile.schema.ts:13]: no .optional() is for undefined and .nullable() is for nullable content

**Review by aaalloc — APPROVED** [review]

**Pull request merged**